core/src/replay_stage.rs

```diff
@@ -439,6 +444,7 @@ impl ReplayStage {
             }
             assert_eq!(*bank_slot, bank.slot());
             if bank.tick_height() == bank.max_tick_height() {
+                did_process_bank = true;
```

Comment:
this clause can eval to true or false obviating the need for a mut var

Reply:
This code is in a loop, how do I avoid the mut var ?

Reply:
missed the loop